xray/src/generation.rs

```diff
@@ -395,7 +395,7 @@ pub fn build_parent(
         child_size_px * 2,
         child_size_px * 2,
         image::Rgba {
-            data: [
+            0: [
```

Comment:
would `image::Rgba([tile_background_color.red, tile_background_color.green, tile_background_color.blue])` work here instead of `image::Rgba{0: [...]}`?
Similarly below.

Reply:
Done